README.md

```diff
@@ -357,3 +357,5 @@ See code for all available configurations.
 | [TUXEDO InfinityBook Pro 14 - Gen7](tuxedo/infinitybook/pro14/gen7)               | `<nixos-hardware/tuxedo/infinitybook/pro14/gen7>`       |
 | [TUXEDO Pulse 14 - Gen3](tuxedo/pulse/14/gen3)                                    | `<nixos-hardware/tuxedo/pulse/14/gen3>`                 |
 | [TUXEDO Pulse 15 - Gen2](tuxedo/pulse/15/gen2)                                    | `<nixos-hardware/tuxedo/pulse/15/gen2>`                 |
+| [Xiaomi (boot fix)](xiaomi/common/fix-boot)                                       | `<nixos-hardware/xiaomi/common/fix-boot>`               |
```

Comment:
It's a bit to vague for me what "some xiaomi" laptops mean.
I would say if we have more models, we just include the module as well.
 
```suggestion
```

Reply:
This workaround might also not be needed in the next hardware generation.